js/test/Ice/idleTimeout/Test.ice

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#pragma once
+
+module Test
+{
+    interface DelayedTestIntf
+    {
+        void sleep(int ms);
+    }
+
+    interface TestIntf extends DelayedTestIntf
+    {
+        void shutdown();
+    }
+
+    interface TestIntfBidir
+    {
+        void putAsSleep(bool aborted, int ms, DelayedTestIntf* target);
```

Comment:
The word is "asleep": https://dictionary.cambridge.org/us/dictionary/english/asleep

But it actually sounds odd with put. So maybe a simpler `makeSleep`?